graspologic/embed/lse.py

```diff
@@ -6,6 +6,8 @@
 from .base import BaseSpectralEmbed
```

Comment:
Could you add type-hinting for everything in this file? Trying to do that because the microsoft people want it in. See  
https://github.com/microsoft/graspologic/pull/772
https://github.com/microsoft/graspologic/pull/543

Reply:
```
regularizer: Optional[float] = None
```
The regularizer can be an int or a float, is this correct?

Reply:
can you link the place in the code where it is

Reply:
https://github.com/microsoft/graspologic/blob/6ced01ec8ddd4e8c417fcd3a9f72b66f12ad39be/graspologic/embed/lse.py#L124